wire-ios/Wire-iOS/Sources/Analytics/TrackingManager/TrackingManager.swift

```diff
@@ -48,21 +48,36 @@ final class TrackingManager: NSObject, TrackingInterface {
             ExtensionSettings.shared.disableAnalyticsSharing
         }
         set {
-            ExtensionSettings.shared.disableAnalyticsSharing = newValue
-            AVSFlowManager.getInstance()?.setEnableMetrics(!newValue)
-
-            do {
-                if newValue {
-                    let disableUseCase = try sessionManager.makeDisableAnalyticsSharingUseCase()
-                    disableUseCase.invoke()
+            self.showAnalyticsConsentAlert { userConsented in
```

Comment:
I don't understand... every time we set the value then we should ask for consent?

Reply:
Fixed it here: https://github.com/wireapp/wire-ios/pull/1767/commits/4d057e135cfeef971c25594ac2e148f186b7e24f